### ISSUE TITLE
What's new in .NET 6 and what's coming in .NET 7

### ISSUE THREAD

**Issue by jowtow** [sfdevs/meetups#29]
**Title**: What's new in .NET 6 and what's coming in .NET 7

**Description**
This would be a discussion on the new features/changes in .NET 6 as well as a larger discussion on the state of the .NET ecosystem and what to expect in .NET 7. I've recently written a few blog posts for Omnitech regarding these topics and our team has been discussing, learning, and using .NET 6 since its release last November.

The Omnitech building is under construction currently, but is slated to have our new addition completed sometime in June. This construction will add on a larger conference room and we figured it'd be a good opportunity to host an SF Devs Meetup there to break in the new facilities. We could probably host this meetup in late June or early July.

I'd be the speaker and I'm sure I'd have some assistance from my fellow Omnitech family.

**Requested Location**
Omnitech

----

- [x]  Speaker: @JohnWTownsend
- [x]  Venue: Omnitech 
- [x]  Date: Thursday, October 27th 
- [x]  Get summary text from speaker
- [x]  Create meetup.com event

**Comment by jxn** [comment]
@JohnWTownsend just to be clear, if we had another venue sooner, would you be interested in that, or do you prefer Omnitech and just prefer to wait for construction to finish?  I'm also a _bit_ wary of construction ever finishing on time, so my vote is to leave a little buffer if you want to use Omnitech for sure.  I'm not _proposing_ another venue as I don't know which we would target, but just asking since that's a way out.

**Comment by jowtow** [comment]
I hear you on the construction timing concern. I think for now I'd want to aim at utilizing Omnitech's new space. There may be other topics brewing from some other folks at Omnitech as well... If more topics come up, we could look at doing this talk sooner with a different venue and breaking in the new conference room with another meetup. 

I'm knocking on wood as I type this, but the construction seems to be coming along as planned so far* 😀

**Comment by jowtow** [comment]
An update on this, our building is still under construction (that's just the way it goes). Hopefully we can get something rolling in late July or early August. Construction seems to be getting pretty close, but we'll want to get familiar with the new space prior to the meetup. I'll update this issue when I know more!

**Comment by jowtow** [comment]
Alright, it's been a minute since I updated this, but we are ready to set a date! I've got my talk all ready to go and right now we are thinking Thursday October 27th around 5:30pm would work well. Omnitech has also agreed to provide food (likely Pizza). 

The topic has also changed a little bit. Instead of targeting .NET 6 and 7 specifically, I'd like to talk more about .NET as a whole. I'm calling the talk "The State of .NET". Towards the end we'll walk through some newer features in more of a demo style. 

**Summary Text:**
The State of .NET
In this session, we'll discuss what .NET is/was/will be. Whether you're new to the .NET ecosystem or have been working with it for years you are more than welcome to attend. We'll also be walking through some of the coolest new features coming in .NET 7 and C# 11! See you there!
